torch/_dynamo/convert_frame.py

```diff
@@ -1200,9 +1200,17 @@ def __call__(
                         user_stack_formatted = "".join(
                             traceback.format_list(user_stack)
                         )
+                        user_stack_trace = (
+                            "Graph break: skip: from user code at:\n%s"
+                            % user_stack_formatted
+                        )
+                        torch._logging.trace_structured(
+                            "dynamo_graph_break_reason",
+                            payload_fn=lambda: "%s\n%s"
+                            % (user_stack_trace, traceback.format_exc()),
```

Comment:
nit: use an f-string, more idiomatic

Reply:
I wonder if there is advantage in keeping `traceback.format_exc()` lazy-evaluated (by the use of `%s`) here - might keep it as-is

Reply:
f-string would still be lazily evaluated, it's the lambda that causes lazy evaluation.....